mcs/class/Mono.Security/Mono.Security/ASN1.cs

```diff
@@ -86,8 +86,8 @@ public ASN1 (byte[] data)
 			Buffer.BlockCopy (data, (2 + nLenLength), m_aValue, 0, nLength);
 
 			if ((m_nTag & 0x20) == 0x20) {
-				int nStart = (2 + nLenLength);
-				Decode (data, ref nStart, data.Length);
+				int nStart = 0;
+				Decode(m_aValue, ref nStart, m_aValue.Length);
```

Comment:
nit: Mono code style should be `Decode (` (with space)

Reply:
fixed